analysis/import/SCMLogParser/src/test/resources/expected_git.json

```diff
@@ -1,35 +1,49 @@
 {
-  "projectName": "SCMLogParser",
-  "apiVersion": "1.0",
-  "nodes": [
-    {
-      "name": "root",
-      "type": "Folder",
-      "attributes": {},
-      "link": "",
-      "children": [
-        {
-          "name": "afolder",
-          "type": "Folder",
-          "attributes": {},
-          "link": "",
-          "children": [
-            {
-              "name": "fileinfolder",
-              "type": "File",
-              "attributes": {
-                "weeks_with_commits": 1,
-                "range_of_weeks_with_commits": 1,
-                "number_of_commits": 2,
-                "successive_weeks_with_commits": 1,
-                "number_of_authors": 1
-              },
-              "link": "",
-              "children": []
-            }
-          ]
-        }
-      ]
-    }
-  ]
-}
\ No newline at end of file
+  "checksum": "foo",
+  "data":{
+    "projectName": "",
```

Comment:
Do you know where the projectName normally comes from?
What is the reason that it is empty now?

Reply:
I did a little research. In https://github.com/MaibornWolff/codecharta/issues/772 the projectName was intended to be removed. But this has never happend (completely). A `DUMMY_PROJECT_NAME` has been introduced which is just an empty string: de.maibornwolff.codecharta.model.ProjectBuilder::DUMMY_PROJECT_NAME

Thus, the empty projectName here is ok.